src/tokens.ts

```diff
@@ -93,6 +93,7 @@ export const TOKEN_SYMBOLS_MAP = {
     symbol: "ETH",
     decimals: 18,
     addresses: {
+      [CHAIN_IDs.ALEPH_ZERO]: "0xD648529D4803d3467bA8850577BEd4e4b8Ae583C",
```

Comment:
Not sure why but seems like we mess up the addresses.
This should be ETH/WETH: https://evm-explorer.alephzero.org/token/0xB3f0eE446723f4258862D949B4c9688e7e7d35d3

```suggestion
      [CHAIN_IDs.ALEPH_ZERO]: "0xB3f0eE446723f4258862D949B4c9688e7e7d35d3",
```